index.html

```diff
@@ -1,12 +1,147 @@
+            Create connections with your users as you engage in genuine discussion.</p>
+          <button class="button2">Get Started For Free</button>
+        </div>
+        <div class="part2">
+          <img src="img/illustration-mockups.svg" alt="imagen">
+        </div>
+      </div>
+    </div>
+    </div>
+    <!-- Fin del primer container -->
+
+    <!-- Comienzo del segundo container -->
+    <div class="container">
+      <div class=" flex cards-containers">
+        <div class="cardtexto">
+          <h2>Grow Together</h2>
+          <p>Generate meaningful discussions with your audience and build a strong, loyal community. Think of the
+            insightful conversations you miss out on with a feedback form. </p>
+        </div>
+
+        <div class="cardimagen">
+          <img src="img/illustration-grow-together.svg" alt="">
+        </div>
+      </div>
+      <!-- Fin del segundo container -->
+
+      <!-- Comienzo del tercer container -->
+      <div class="flex cards-containers">
+        <div class="cardtexto">
+          <h2>Flowing Conversations</h2>
+          <p>You wouldn't paginate a conversation in real life, so why do it online? Our threads have just-in-time
+            loading for a more natural flow.</p>
+        </div>
+        <div class="cardimagen">
+          <img src="img/illustration-flowing-conversation.svg" alt="">
+        </div>
+      </div>
+      <!-- Fin del tercer container -->
+
+      <!-- Comienzo del cuarto container -->
+      <div class="flex cards-containers">
+        <div class="cardtexto">
+          <h2>Your Users</h2>
+          <p>It takes no time at all to integrate Huddle with your app's authentication solution. This means, once
+            signed in to your app, your users can start chatting immediately.</p>
+        </div>
+        <div class="cardimagen">
+          <img src="img/illustration-your-users.svg" alt="">
+        </div>
+      </div>
+      <!-- Fin del cuarto container -->
+
+
+      <div class="cards-containers cardfinal">
+        <h3>Ready To Build Your Community?</h3>
+        <button class="buttonfinal"> Get Started For Free </button>
+      </div>
+    </div>
+
+  </section>
+  <!-- Fin de los containers -->
+
+          <!-- Inicio Footer -->
+  <footer class="footer">
+    <div class="container">
+      <img src="img/logo.svg" alt="Logo" class="logo-footer">
+      <div class="footer-contact-info footer-flex">
+        <div class="footer-div">
+          <div class="footer-innerdiv">
+            <img src="img/icon-location.svg" alt="" class="footer-icons">
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et dolore magna aliqua</p>
+          </div>
+
+          <div class="footer-innerdiv">
+            <img src="img/icon-phone.svg" alt="" class="footer-icons">
+            <p>+1-543-123-4567</p>

```

Comment:
Es un numero de telefono
```suggestion
            <a href="tel:+15431234567">+1-543-123-4567</a>
```